README.md

```diff
@@ -3,6 +3,38 @@
 A companion plugin for [nvimtools/none-ls.nvim](https://github.com/nvimtools/none-ls.nvim),
 adding support for [cspell] diagnostics and code actions.
 
+## How to setup in lazyvim
+
+Make sure you have `cpsell` installed, and then add the following
+to your plugin configuration directory.
+
+```lua
+return {
+  {
+    "davidmh/cspell.nvim",
+    event = "VeryLazy",
+  },
```

Comment:
We don't really need to list the plugin as it's own entry, being listed in the none-ls dependencies is enough to have lazyvim install it.